<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_equipment</class>
 <widget class="QMainWindow" name="delete_equipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Удаление оборудования</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTableView" name="tableView_show_item"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_show">
      <property name="text">
       <string>Показать в таблице</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_invent_num">
      <property name="text">
       <string>Инвентарный номер</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QLineEdit" name="invent_num"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_part_num">
      <property name="text">
       <string>Артикул</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="part_num"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_delete">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
